src/Parsing/Impl/Tokens/Tokenizer.cs

```diff
@@ -111,6 +111,12 @@ public sealed class Tokenizer {
             return tokens;
         }
 
+        /// <summary>
+        /// Current state, to be passed to <see cref="Initialize(object, TextReader, SourceLocation)"/>
+        /// or <see cref="Initialize(object, TextReader, SourceLocation, int)"/>./>
```

Comment:
Can you make a note about why this is needed in PTVS so we don't accidentally remove it again? (Sorta like what you did in #1541.)